modules/kubernetes/aks-core/modules.tf

```diff
@@ -238,10 +238,11 @@ module "ingress_healthz" {
 
   source = "../../kubernetes/ingress-healthz"
 
-  environment     = var.environment
-  dns_zone        = var.cert_manager_config.dns_zone[0]
-  location_short  = var.location_short
-  linkerd_enabled = var.linkerd_enabled
+  environment            = var.environment
+  dns_zone               = var.cert_manager_config.dns_zone[0]
+  location_short         = var.location_short
+  linkerd_enabled        = var.linkerd_enabled
+  public_private_enabled = var.ingress_config.public_private_enabled
```

Comment:
Do we think it's a good idea to use this config variable that should be used for ingress-nginx?